tests/test_installation.py

```diff
@@ -183,5 +189,79 @@ def test_pico_migrate(self):
             'city.yml', 'graph.xml', 'namespaces.txt']))
 
 
+class PicoModule(unittest.TestCase):
+    """Test the use of pico as a Python module."""
+
+    def setUp(self) -> None:
+        """Change installation path and reset the namespace registry."""
+        self._previous_installation_path = OntologyInstallationManager\
+            .get_default_installation_path()
+        self._new_installation_path = \
+            pathlib.Path('.TEST_OSP_CORE_INSTALLATION').absolute()
+        os.makedirs(self._new_installation_path / '.osp_ontologies',
+                    exist_ok=True)
+        OntologyInstallationManager.set_default_installation_path(
+            str(self._new_installation_path))
+        namespace_registry.clear()
+        namespace_registry.load_graph_file(
+            OntologyInstallationManager.get_default_installation_path())
+
+    def tearDown(self) -> None:
+        """Revert changes done during the execution of the `setUp` method."""
+        OntologyInstallationManager.set_default_installation_path(
```

Comment:
Same here, how do you know the default was the one used before?

Reply:
No custom path given [here](https://github.com/simphony/osp-core/blob/9fbfe4dee86dfbf4494593e0cdc5cfd5194dab2e/osp/core/pico.py#L18). Pico uses such `OntologyInstallationManager`.